src/OrleansPSUtils/OrleansPSUtils.2017.csproj

```diff
@@ -0,0 +1,59 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup Label="NuGet">
+    <IsPackable>true</IsPackable>
+    <PackageId>Microsoft.Orleans.OrleansPSUtils</PackageId>
```

Comment:
Nuget package for PSUtils? Unless you plan to make something like `dotnet ef` which is a *tool* package that is distributed by Nuget packages but only work inside the project directory context (i.e. not a global tool), I don't see why you would need that.

As far as I know, the only official PS Cmdlet distribution model is thru https://www.powershellgallery.com/ and unofficially, from Chocolatey.

Can you elaborate more on why you need a Nuget for it? The gallery deployment is something I was going to arrange with @jdom a while ago and make it pass thru the same sign process that other packages goes thru.